lms/static/js/spec/student_profile/learner_profile_fields_spec.js

```diff
@@ -0,0 +1,276 @@
+define(['backbone', 'jquery', 'underscore', 'js/common_helpers/ajax_helpers', 'js/common_helpers/template_helpers',
+        'js/spec/student_account/helpers',
+        'js/student_account/models/user_account_model',
+        'js/student_profile/views/learner_profile_fields',
+        'js/views/message_banner'
+       ],
+    function (Backbone, $, _, AjaxHelpers, TemplateHelpers, Helpers, UserAccountModel, LearnerProfileFields,
+              MessageBannerView) {
+        'use strict';
+
+        describe("edx.user.LearnerProfileFields", function () {
+
+            var createImageView = function (ownProfile, hasImage, imageMaxBytes, imageMinBytes, yearOfBirth) {
```

Comment:
Nit: I find code with this many parameters to be hard to read, as it isn't clear which value is which. I would suggest taking an options object, and then the caller can clearly indicate which option they are setting, e.g. yearOfBirth.